gpytorch/functions/_pivoted_cholesky.py

```diff
@@ -98,7 +98,11 @@ def backward(ctx, grad_output, _):
 
         with torch.enable_grad():
             # Create a new set of matrix args that we can backpropagate through
-            matrix_args = [matrix_arg.detach().requires_grad_(True) for matrix_arg in _matrix_args]
+            matrix_args = []
```

Comment:
I would say we could also catch the error so that if there are any future changes we don't have to adjust the code here. But then raising errors is unfortunately quite slow in ptorch...